solarnet/peering.sh

```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 
-# Builds a cjdroute.conf snippet for peering with all nodes in secrets.yml
-#
-# Usage:
-#   ./peering.sh
+function usage {
+  echo "Builds a cjdroute.conf snippet for peering with the specified ansible hosts, using the specified password"
+  echo
+  echo "Usage:"
+  echo "  ./peering.sh <pattern> <password-name>"
+  echo "  ./peering.sh all solarnet"
+  echo "  ./peering.sh gateway alexandria"
```

Comment:
Sorry, I was under the impression you're aware of alexandria -- they're using IPFS for storage and distribution, along with a few altcoins. Ryan gave a brief presentation at Battlemesh.

gendale_ on IRC asked for peering with our gateway nodes.

> I think its just an example hes using for the usage text? Although the title of the PR makes me think it might be otherwise

It's meant to serve just as an example, although there are indeed peering passwords named solarnet and alexandria in the private secrets.yml.